app/Console/Kernel.php

```diff
@@ -7,19 +7,6 @@
 
 class Kernel extends ConsoleKernel
 {
-    /**
-     * The Artisan commands provided by your application.
-     *
-     * @var array
-     */
-    protected $commands = [
```

Comment:
Did you intend to move these to the `routes/console.php`? Or deleted altogether?

Reply:
I removed it because the auto-discovery is in play AND i didn't want to null the array that discovery makes.

Reply:
Sounds good.